src/store/trees/trees.js

```diff
@@ -345,6 +346,7 @@ export default {
     },
 
     addSubscriberPropertyField({ ui }, { property }) {
+      console.log('addSubscriberPropertyField', ui)
```

Comment:
Should this console.log be commited? If so, maybe it should be `info`?

Reply:
nop, this was a dev debug. Removing it. thanks